src/metabase/query_processor/store.clj

```diff
@@ -178,8 +178,23 @@
   [database-id-or-metadata-provider & body]
   `(do-with-metadata-provider ~database-id-or-metadata-provider (^:once fn* [] ~@body)))
 
+(defn- pretty-metadata-error-msg
```

Comment:
This name suggests more than it can actually do. It can only report problems related to failed lookups.